packages/cdktf-cli/src/bin/cmds/convert.ts

```diff
@@ -50,12 +50,25 @@ class Command extends BaseCommand {
         type: "boolean",
         default: false,
       })
+      .option("without-project", {
+        describe:
+          "Convert creates a temporary project by default on non-Typescript languages. Use this flag to disable that behavior. Note: Disabling this will cause conversion to be less correct",
+        type: "boolean",
+        default: false,
+      })
       .showHelpOnFail(true);
 
   public async handleCommand(argv: any) {
     Errors.setScope("convert");
     // deferred require to keep cdktf-cli main entrypoint small (e.g. for fast shell completions)
     const api = requireHandlers();
+    if (argv.withProject !== undefined) {
```

Comment:
I think most users won't need it, but I was thinking about the registry work where we don't want this behavior per-conversion. If we're not using it there, I'm happy to remove the flag.

Reply:
That's a good point. How about a default value in the call to hcl2cdk? The registry tool uses the hcl2cdk package so it can overwrite the default value :)